### ISSUE TITLE
MySQL password should not be kept in repo

### ISSUE THREAD

**Issue by paulcmal** [pierreozoux/hosting#1]
In [recreate_mysql_root_user](https://github.com/pierreozoux/hosting/blob/master/procedure/recreate_mysql_root_user) there is a password hanging around. Probably it's not the one you use in production but just in case i wanted to let you know :)

**Comment by pierreozoux** [comment]
Yeah, I'm even wondering what is this repo :) I'll archive it, but thanks for the hint!
